web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -63,6 +61,22 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
                 },
                 getMiniMapLinkTitle: function(values) {
                     return (this.isRecActive(values.uuid)) ? OpenLayers.i18n('collectionExistsMsg') : OpenLayers.i18n("addDataCollectionMsg");
+                },
+                getTitle: function(values) {
+                    var broader = [];  
+                    Ext.each(values.parameter, function(param) {
+                        var broaderTerms = this.classificationStore.getBroaderTerms(param, 2, 'Measured parameter');
+                        if(broaderTerms.length > 0) { 
+                            broader = broader.concat(broaderTerms); 
+                        }
+                        else {
+                            console.log("Could not find broader term for parameter '" + param  + "'");
```

Comment:
I think I'll comment them. Just to keep a record - these are the missing parameter/platform terms.

window.controllers is deprecated. Do not use it for UA detection. OpenLayers.js:1171
"Could not find broader term for parameter 'Speed (over ground) of measurement platform'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Direction of motion (over ground) of measurement platform'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Orientation (pitch) of measurement platform'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Orientation (roll angle) of measurement platform'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Direction of motion (over ground) of measurement platform'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Height above sensor'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Depth below surface of the water body'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Correlation in magnitude of the particle distribution in the water body'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Percentage of good three beam solutions (one beam rejected) in ADCP bin'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Percentage of good transformations (error velocity threshold not exceeded) in ADCP bin'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Percentage of measurements where more than one beam was bad in ADCP bin'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Percentage of measurements with four beam solutions in ADCP bin'" FacetedSearchResultsDataView.js:73
"Could not find broader term for parameter 'Depth below surface of the water body'" FacetedSearchResultsDataView.js:73
"Could not find broader term for platform ''" FacetedSearchResultsDataView.js:172
"Could not find broader term for parameter 'Depth below surface of the water body'" FacetedSearchResultsDataView.js:73